includes/class-sensei-quiz.php

```diff
@@ -99,8 +99,15 @@ public function is_block_based_editor_enabled() {
 
 		$current_screen = function_exists( 'get_current_screen' ) ? get_current_screen() : null;
 
+		$is_block_editor = (
+			! $current_screen || $current_screen->is_block_editor()
+		) || (
+			// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- Don't touch the nonce.
+			isset( $_GET['meta-box-loader-nonce'] ) && wp_verify_nonce( wp_unslash( $_GET['meta-box-loader-nonce'] ), 'meta-box-loader' )
```

Comment:
Notice that this query string is used to save the meta box through this request: https://github.com/WordPress/gutenberg/blob/0592a37671b6605048590bd9c52f6b8f47f8b427/packages/edit-post/src/store/actions.js#L581 (coming from the editor), so I assume that it's safe to say that `$is_block_editor` is `true` in this case.